src/pages/patientView/PatientViewPage.tsx

```diff
@@ -85,19 +88,44 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
 
     @observable private mutationTableColumnVisibility: {[columnId: string]: boolean}|undefined;
     @observable private cnaTableColumnVisibility: {[columnId: string]: boolean}|undefined;
+    
+    // use this wrapper rather than interacting with the url directly 
+    private urlWrapper: PatientViewUrlWrapper
 
     constructor(props: IPatientViewPageProps) {
 
         super(props);
+        this.urlWrapper = new PatientViewUrlWrapper(props.routing);
 
+        if (
+            this.props.routing.location.pathname.includes("/" + PagePath.Patient) &&
+            this.urlWrapper.query.studyId
+        ) {
+            trackPatient(this.urlWrapper.query.studyId);
+        }
+        
+        reaction(
+            () => [this.studyIdAndCaseId],
+            ([_]) => {
+                if (
+                    this.props.routing.location.pathname.includes("/" + PagePath.Patient) &&
```

Comment:
how come we do this test?

Reply:
I saw this test further down when the routing store is populated from the url. The logic there was that you didn't want to do that stuff if this isn't a patient view. Along the same lines, I felt that if this wasn't a patient view, we didn't want to log that we had visited a patient.